src/test/java/fiori/steps/ServiceOrderQuotationSteps.java

```diff
@@ -0,0 +1,34 @@
+package fiori.steps;
+
+import eu.ibagroup.junase.model.util.Assert;
+import fiori.screens.ServiceOrderQuotationScreen;
+import fiori.util.TextUtil;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class ServiceOrderQuotationSteps {
+
+    private final ServiceOrderQuotationScreen serviceOrderQuotationScreen = new ServiceOrderQuotationScreen();
+
+    @When("^I set (Description) as random string on tab (Quotation Details) on screen (Service Order Quotations) in application (Fiori)$")
+    public void setInputByLabelAsRandomValue(String labelName, String tab, String screen, String app) {
+        String randomValue = TextUtil.generateAlphanumericString(11);
```

Comment:
please use the built-in method from the class eu.ibagroup.junase.model.util.TextUtil